packages/react/src/components/Table/TableHead/TableHead.jsx

```diff
@@ -669,6 +669,7 @@ const TableHead = ({
             isFilterable: !isNil(column.filter),
             isMultiselect: column.filter?.isMultiselect,
             width: column.width,
+            customInput: column?.customInput,
```

Comment:
I would imagine we would want this to be part of the filter object. Like above on line 670,